src/tests/particles/particles-test.ts

```diff
@@ -4,14 +4,8 @@ import glob from 'glob';
 import { Loader } from '../../runtime/loader.js';
 
 // Skip these files, they do not compile successfully.
-// TODO: Figure out why.
 const FILES_TO_SKIP = new Set([
```

Comment:
Can we remove this variable and comment now?

Reply:
I thought to leave that decision to @csilvestrini, but I can amend the PR as needed.

Reply:
No problemo.